eol_forum_notifications/management/commands/discussion_notification.py

```diff
@@ -23,8 +23,6 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         logger.info('EolForumNoticationsCommand - Running send_notification()')
-        if not options['how_often']:
-            raise CommandError("EolForumNoticationsCommand - how_often must be specified")
         if options['how_often'] not in ['weekly', 'daily']:

```

Comment:
esto no se cae si `options['how_often']` is None? puede `options` venir indefinido? es realmente inalcanzable? o es sólo inalcanzable en el contexto de los tests?